scripts/ci/gitlab/pipeline/test.yml

```diff
@@ -58,6 +58,18 @@ cargo-fmt:
     - cargo +nightly fmt --all -- --check
     - rusty-cachier cache upload
 
+cargo-fmt-manifest:
+  stage: test
+  extends:
+    - .docker-env
+    - .test-refs
+  script:
+    - cargo install zepter --locked --version 0.10.0 -q -f --no-default-features && zepter --version
```

Comment:
```suggestion
```

I'll go and include zepter in our CI docker image so it doesn't need to be installed over and over again by every CI run.

Reply:
Yep i would first like to stabilize the check a bit - it is still allowed to fail.